blivet/devices/lvm.py

```diff
@@ -1471,6 +1475,13 @@ def vg_space_used(self):
             space += Size(blockdev.lvm.get_thpool_padding(space, self.vg.pe_size))
         return space
 
+    def _pre_create(self):
+        if self._from_lvs:
+            # make sure all the LVs this LV should be created from exist
+            for lv in self._from_lvs:
+                if not lv.exists:
+                    lv.create()
```

Comment:
Normally, such actions should be scheduled and the component LVs should already be created when this code is executed. The creation of the component LVs should only happen here as a last resort. E.g. in cases where some code doesn't use actions (like tests).